API/DTOs/TournamentCompactDTO.cs

```diff
@@ -0,0 +1,77 @@
+using System.Diagnostics.CodeAnalysis;
+using Database.Enums;
+using Database.Enums.Verification;
+
+namespace API.DTOs;
+
+public class TournamentCompactDTO
+{
+    /// <summary>
+    /// The tournament id
+    /// </summary>
+    public int Id { get; init; }
+
+    /// <summary>
+    /// The name of the tournament
+    /// </summary>
+    public string Name { get; init; } = null!;
+
+    /// <summary>
+    /// Acronym / shortened name of the tournament
```

Comment:
```suggestion
    /// Acronym / shortened name
```
The docs for `TournamentDTO` are pretty old and we have shifted away from using phrases like `<description of prop> for the <entity name>`. If we could update the rest of these here that would be great